test/Customer-test.js

```diff
@@ -1,50 +1,32 @@
 import chai from 'chai';
 const expect = chai.expect;
+import Customer from '../src/classes/Customer';
+const customers = require('../src/sampleData/customersData')
 
 describe('Customer', () => {
+  let customer1, customer2, customer3;
+
+  beforeEach(() => {
+    customer1 = new Customer(customers[0].id, customers[0].name);
+    customer2 = new Customer(customers[1].id, customers[1].name);
+    customer3 = new Customer(customers[2].id, customers[2].name);
+
+  });
 
   it('Should be a function', () => {
     expect(Customer).to.be.a('function');
   });
 
   it('Should be an instance of Customer', () => {
-    expect(customer).to.be.an.instanceOf(Customer);
+    expect(customer1).to.be.an.instanceOf(Customer);
   });
 
```

Comment:
I suggest adding another test for "should be able to instantiate another instance of Customer"

Reply:
Thanks for all the feedback! I'll take those suggestions into account and apply. :)